cmd/create/cluster/cmd.go

```diff
@@ -693,6 +698,27 @@ The password must
 			strings.Join(ingress.ValidNamespaceOwnershipPolicies, ","), ingress.DefaultNamespaceOwnershipPolicy),
 	)
 
+	flags.StringVar(
+		&args.privateHostedZoneID,
+		"private-hosted-zone-id",
+		"",
+		`ID of shared private hosted zone`,
+	)
+
+	flags.StringVar(
+		&args.privateHostedZoneRoleARN,
+		"private-hosted-zone-role-arn",
```

Comment:
I wonder if we should stick with the naming convention we have in `create operator-roles` which is `--shared-vpc-role-arn` because it's the same value in both commands.